container.go

```diff
@@ -68,126 +64,10 @@ func getCRIOInfo(_ *metadata.ContainerConfig, specDump *spec.Spec) (*containerIn
 }
 
 func showContainerCheckpoints(tasks []task) error {
-	// Return an error early when attempting to display multiple checkpoints with additional flags
-	if (len(tasks) > 1) && (mounts || stats || psTree) {
-		return fmt.Errorf("displaying multiple checkpoints with additional flags is not supported")
+	if format == "table" {
+		return renderTableView(tasks)
```

Comment:
It might be good to use a `switch` statement here and show an error when the value of `format` is invalid.